### PR TITLE
Added profile picture and fixed bug

### LINKED ISSUE #60: Profile Picture not getting displayed on receiver side.
The profile picture is displayed on the sender's end, but not getting displayed on the receiver's end.

ss

As you can see I have uploaded DP on the left side, It's getting uploaded to Firestore also, but not getting displayed on the emulator  that's on the right side

### PR DESCRIPTION
**Pull request by riturajjain2000**
Fixes #60 #59

### CODE REVIEW

File: app/src/main/res/layout/activity_chat.xml [hunk 1]
```diff
@@ -1,66 +1,144 @@
...
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:background="@drawable/wallpaper"
     tools:context=".Activity.ChatActivity">
 
+    <androidx.appcompat.widget.Toolbar
```
Comment (plazzy99) [inline app/src/main/res/layout/activity_chat.xml:11]: You can also do this without adding toolbar manually  just giving an icon to the default toolbar

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -4,6 +4,7 @@
     package="com.vatsal.kesarwani.therapy">
 
     <uses-permission android:name="android.permission.CALL_PHONE" />
+    <uses-permission android:name="android.permission.RECORD_AUDIO" />
```
Comment (plazzy99) [inline app/src/main/AndroidManifest.xml:7]: no need to add this for this pr

**Review by plazzy99 — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/activity_chat.xml (hunk 1 above)
Comment (plazzy99) [reply app/src/main/res/layout/activity_chat.xml:11]: just a comment

File: app/src/main/AndroidManifest.xml (hunk 2 above)
Comment (plazzy99) [reply app/src/main/AndroidManifest.xml:7]: resolve it

File: app/src/main/res/layout/activity_chat.xml (hunk 1 above)
Comment (riturajjain2000) [reply app/src/main/res/layout/activity_chat.xml:11]: I tried but that's not possible with the original Actionbar

File: app/src/main/AndroidManifest.xml (hunk 2 above)
Comment (riturajjain2000) [reply app/src/main/AndroidManifest.xml]: Done!

**Review by plazzy99 — APPROVED** [review]
👍🏻

**Pull request merged**